packages/state-transition/test/perf/epoch/epochAltair.test.ts

```diff
@@ -6,7 +6,8 @@ import {
   CachedBeaconStateAltair,
   beforeProcessEpoch,
 } from "../../../src/index.js";
-import {getNetworkCachedState, beforeValue, LazyValue} from "../../utils/index.js";
+import {beforeValue, LazyValue} from "../../utils/beforeValueMocha.js";
```

Comment:
Why is this file called `beforeValueMocha.js` aren't we getting rid of mocha?

Reply:
Explained in the comment below.